index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
```

Comment:
Please use descriptive meta tags in the head section, this will impact the way your site appears in the SERPs and how many people will be inclined to click through to your website. Thereby impacting your traffic and engagement rates, which can impact your SEO and rankings.